crates/meilisearch/src/routes/indexes/search_analytics.rs

```diff
@@ -1,18 +1,16 @@
-use once_cell::sync::Lazy;
```

Comment:
The CI was failing because of `cargo fmt`...
I don’t know why I was able to merge the analytics PR initially, but something in this PR was triggering the CI and blocking me 😭

Reply:
understood